### ISSUE TITLE
"New Command/Entry/Item" translation issue

### ISSUE THREAD

**Issue by lazka** [quodlibet/quodlibet#4100]
Pointed out on weblate: https://github.com/quodlibet/quodlibet/blob/master/quodlibet/qltk/data_editors.py#L117

> This approach doesn’t work for gendered languages. In Spanish this would produce “Nuevo entrada”, which is obviously incorrect.

**Comment by declension** [comment]
Yes - but not sure what the solution is. I guess we can iterate the known combinations somewhere to force them to get picked up by gettext, but they still need to be dynamic at runtime. I guess maybe a simple class hierarchy would do it.
